texspell.sh

```diff
@@ -314,6 +309,14 @@ function keep_x_last_words {
   echo $(echo "$1" | tr ' ' '\n' | tail -${2} | xargs -n${2})
 }
 
+# Create a new tmp file 
+# 1 filename
+#
+# R - path tho file
+function create_file {
+  echo $(mktemp -p "${TEMP_DIR}" "$13_XXXXX.tmp")
```

Comment:
```suggestion
  echo $(mktemp -p "${TEMP_DIR}" "${1}_XXXXX.tmp")
```

I guess you meant that ?